scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -49,11 +49,17 @@
 {% endmacro %}
 
 
-{% macro callers_cell(callers) %}
-  {% for name, caller in callers %}
+{% macro callers_cell(variant) %}
+  {% for name, caller in variant.callers %}
       <span class="badge {% if caller == 'Pass' %}bg-success{% elif caller == 'Filtered' %}bg-secondary{% else %}bg-black{% endif %}" data-bs-toggle="tooltip" data-bs-html="true" title="{{caller}}">
         {{ name }}
       </span>
+  {% else %} <!-- collect info from variant's filters -->
+    {% for filter in variant.filters %}
+      <span class="badge bg-{{ filter.label_class }}" data-bs-toggle="tooltip" data-bs-placement="top" title="{{ filter.description }}">
+        {{ filter.label }}
```

Comment:
On a cursory look, the filters are quite a bit richer, and has nicely templated categories. I feel when we have them we should show them? But, let's check a few files or discuss with the Balsamic team I guess!